galite-core/src/main/kotlin/org/kopi/galite/report/VReport.kt

```diff
@@ -567,7 +563,7 @@ abstract class VReport internal constructor(ctxt: DBContextHandler? = null) : VW
   fun genHelp(): String? {
     val surl = StringBuffer()
     val fileName: String? = VHelpGenerator().helpOnReport(pageTitle,
-            commands,
+            commands?.requireNoNulls(),
```

Comment:
Should have changed : 
var commands: Array<VCommand?>? to array of not nulls ,  instead of using .requireNoNulls()